components/server/ee/src/user/linkedin-service.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { ErrorCodes } from "@gitpod/gitpod-protocol/lib/messaging/error";
+import { inject, injectable } from "inversify";
+import fetch from "node-fetch";
+import { ResponseError } from "vscode-jsonrpc";
+import { Config } from "../../../src/config";
+
+@injectable()
+export class LinkedInService {
+    @inject(Config) protected readonly config: Config;
+
+    async getAccessToken(code: string) {
+        const { clientId, clientSecret } = this.config.linkedInSecrets || {
+            clientId: "86lcfbj2vwm2ba", // FIXME(janx)
```

Comment:
We need to cycle both of these config values since they've been committed to the repo.

Reply:
Actually just the client secret. The client ID is public information. The secret is supposed to be a secret, but for faster development I've committed it in cleartext, and I'll rotate it once we put in an actual secret before shipping this 👍

Reply:
Done